quickwit/quickwit-serve/src/elastic_search_api/model/search_body.rs

```diff
@@ -222,4 +223,25 @@ mod tests {
         assert_eq!(field_sorts[3].field, "_doc");
         assert_eq!(field_sorts[3].order, SortOrder::Asc);
     }
+
+    #[test]
+    fn test_unknown_field_behaviour() {
+        let json = r#"
+            {
+                "term": {
+                    "actor.id": {
+                        "value": "95077794"
+                     }
+                }
+            }
+        "#;
+
+        let search_body = serde_json::from_str::<SearchBody>(json);
+        assert!(search_body.is_err());
+        assert_eq!(
+            search_body.err().unwrap().to_string(),
+            "unknown field `term`, expected one of `from`, `size`, `query`, `sort`, `aggs`, \
```

Comment:
can we make the assert less brittle?

Check that the error message contains a few keyword "unknown field" for instance

Reply:
sure, thanks!